jc/parsers/systemextensionsctl.py

```diff
@@ -0,0 +1,165 @@
+"""jc - JSON Convert `systemextensionsctl list` command output parser
+
+Usage:
+
+    $ systemextensionsctl list | jc --systemextensionsctl
+
+Compatibility:
+
+    macOS
+
+Example:
+
+    $ systemextensionsctl list | jc --systemextensionsctl -p
+    {
+      "total_extensions": 1,
+      "sections": [
+        {
+          "category": "com.apple.system_extension.network_extension",
+          "description": "Go to 'System Settings > General > Login Items & Extensions > Network Extensions' to modify these system extension(s)",
+          "entries": [
+            {
+              "enabled": "*",
+              "active": "*",
```

Comment:
Looking good! We'll need to add the schema here. I wonder if `enabled` and `active` should be boolean?